app/components/wds-global-navigation.js

```diff
@@ -31,6 +41,14 @@ export default Component.extend({
 		assert('Required property `model` is not set', this.model);
 	},
 
+	getRedirectUrl() {
```

Comment:
Maybe this could be a computed property based on 'fastboot.isFastBoot'? Then it would recalculate automatically both on fastboot and front-end.